### DISCUSSION TITLE
1/2

### DISCUSSION THREAD

**Discussion by BeMatthewsong** [Team-Plants/PLANTS#117] [Daily Scrum]
## 김윤후

- 오늘의 컨디션 :  8점
- 어제 진행했거나 완료한 작업 : 대시보드 삭제 구현, 민혁님 도와드림
- 오늘 진행할 작업 : 마무리 작업
- 막혔던 부분 : 메인페이지 로그인되어있으면 리다이렉트 처리
- 도움이 필요한 부분 : x
- TMI : 2시 반에 위워크에 도착할 것 같다

## 송민혁

- 오늘의 컨디션 : 6점 
- 어제 진행했거나 완료한 작업 : 무한스크롤, 카드 수정
- 오늘 진행할 작업 : 마무리 작업
- 막혔던 부분 : pr에 다 써놓음
- 도움이 필요한 부분 :  많습니다...
- TMI : 떡국 두 그릇이나 먹었습니다

## 안혜정

- 오늘의 컨디션 : 7점 
- 어제 진행했거나 완료한 작업 : 헤더
- 오늘 진행할 작업 : 무한스크롤, 마무리 도움
- 막혔던 부분 : x
- 도움이 필요한 부분 : 무한스크롤
- TMI : 갑진년 기념 다이어리 ~~~ 

## 전수빈

- 오늘의 컨디션 : 3점 
- 어제 진행했거나 완료한 작업 : 에러핸들링 (토큰, api 잘못된 요청 시)
- 오늘 진행할 작업 : css, 다른 분들 도와줄 예정
- 막혔던 부분 : x
- 도움이 필요한 부분 : x
- TMI : 다이어리 살까말까

## 한슬희

- 오늘의 컨디션 : 3점
- 어제 진행했거나 완료한 작업 : 계정 API
- 오늘 진행할 작업 :  계정 API 마무리
- 막혔던 부분 : 새로고침
- 도움이 필요한 부분 : 새로고침
- TMI : 2시간 잠...

**Comment by K-Y-Hoo** [comment]
저도 다이어리 하나 장만해야겠어요~ ^^ ~
